src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -830,10 +830,10 @@ type TaskSeq =
     static member takeWhile: predicate: ('T -> bool) -> source: TaskSeq<'T> -> TaskSeq<'T>
 
     /// <summary>
-    /// Returns a sequence that, when iterated, yields elements of the underlying sequence while the
+    /// Returns a task sequence that, when iterated, yields elements of the underlying sequence while the
     /// given asynchronous function <paramref name="predicate" /> returns <see cref="true" />, and then returns no further elements.
     /// The first element where the predicate returns <see cref="false" /> is not included in the resulting sequence
-    /// (see also <see cref="TaskSeq.takeWhileInclusive" />).
+    /// (see also <see cref="TaskSeq.takeWhileInclusiveAsync" />).
     /// If <paramref name="predicate" /> is synchronous, consider using <see cref="TaskSeq.takeWhile" />.
```

Comment:
consider using -> use/see

Reply:
Hmm, you mean:

```html
If <paramref name="predicate" /> is synchronous, use <see cref="TaskSeq.takeWhile" />.
```

I thought we removed that long time ago, as it was too commanding? I like shorter, but then we should do it everywhere, as 50% of the functions here have this text snippet. I'll keep it in mind for another doc refactoring.